### ISSUE TITLE
readmeio-sync upload not fetching all versions

### ISSUE THREAD

**Issue by jennifer-shehane** [haroldtreen/readmeio-sync#15]
I've setup my Readme.io docs with 2 versions. "v0.0" and "v1.0". The initialization works fine when pulling the project from readme.io. 

`reademeio-sync init`
<img width="395" alt="screen shot 2016-01-12 at 3 14 03 pm" src="https://cloud.githubusercontent.com/assets/1271364/12275687/6f7b3c6c-b93f-11e5-9d7a-c0c82aa4c32a.png">

But when I run upload (even after making zero changes to the files), it does not appear to fetch both versions and eventually errors out. 

`readmeio-sync upload`
<img width="323" alt="screen shot 2016-01-12 at 3 14 16 pm" src="https://cloud.githubusercontent.com/assets/1271364/12275721/a387de34-b93f-11e5-9268-0f68b91918fe.png">

`SyntaxError: Unexpected token <` after "Creating documentation categories..."

**Comment by haroldtreen** [comment]
Thanks for reaching out @jennifer-mann .

Have you added the new version to your `syncPaths.json`?
If you add a new project on the site, but don't specify the path to a local copy that could cause a problem...

**Comment by jennifer-shehane** [comment]
Yes, the versions are in the `syncPaths.json`:

``` json
{
    "cypress": {
        "v0.0": {
            "documentation": "cypress/v0.0/documentation",
            "customPages": "cypress/v0.0/customPages",
            "customContent": {
                "appearance": {
                    "html_body": "cypress/v0.0/customContent/appearance/landing_page.html",
                    "stylesheet": "cypress/v0.0/customContent/appearance/styles.css"
                }
            }
        },
        "v1.0": {
            "documentation": "cypress/v1.0/documentation",
            "customPages": "cypress/v1.0/customPages",
            "customContent": {
                "appearance": {
                    "html_body": "cypress/v1.0/customContent/appearance/landing_page.html",
                    "stylesheet": "cypress/v1.0/customContent/appearance/styles.css"
                }
            }
        }
    }
}
```

**Comment by jennifer-shehane** [comment]
I figured this out. I actually have 2 projects on readme.io that I have specified in my `syncConfig.json`

``` json
{
    "projectNames": {
        "staging": "cypress-staging",
        "production": "cypress"
    }
}
```

The `init` was pulling in data from`cypress`, but the `upload` was going to my `cypress-staging`. I had never created a version `0.0` on the cypress-staging project, so it would error out (albeit in an inelegant fashion). It all works fine after adding a `0.0` version to both projects.
